source/index.md

```diff
@@ -5759,6 +5845,92 @@ The admin can list all SoftLayer Cloud Server plans accessing the route bellow.
 
 Route: *`GET "/api/v1/admin/shop-softlayer-cloud-server-plans"`*
 
+# Admin Shop Price
+
+When logged in, the admin can access information about the shop. This access allows him to create shop prices.
```

Comment:
Same ideas applied to users (do not create and plan attributes), can be used on admin.